apps/test/netsim/NetSimConnection.js

```diff
@@ -24,7 +24,7 @@ describe("NetSimConnection", function () {
     testWindow = {
       addEventListener: function () {}
     };
-    connection = new NetSimConnection(testWindow, null, null, false);
+    connection = new NetSimConnection(testWindow, {}, null, null, false);
```

Comment:
i think this many parameters means you're getting close to better passing a hash with named parameters.